File: PhotoBank.Dto/View/PhotoDto.cs
using System;
using System.Collections.Generic;

namespace PhotoBank.Dto.View
{
    public class PhotoDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Scale { get; set; }
        public DateTime? TakenDate { get; set; }
        public byte[] PreviewImage { get; set; }
        public int? Orientation { get; set; }
        public List<FaceDto> Faces { get; set; }
        public List<string> Captions { get; set; }
        public List<string> Tags { get; set; }
        public double AdultScore { get; set; }
        public double RacyScore { get; set; }
        public int Height { get; set;}
        public int Width { get; set; }
    }
}

File: Photobank.Ts/packages/shared/src/types/dto/PhotoDto.ts
import { FaceDto } from './FaceDto';

export interface PhotoDto {
  id?: number;
  name: string;
  scale?: number;
  takenDate?: string;
  previewImage: string;
  orientation?: number;
  faces?: FaceDto[];
  captions?: string[];
  tags?: string[];
  adultScore?: number;
  racyScore?: number;
  height?: number;
  width?: number;
}